jekyll/_cci2/linux-cuda-images-support-policy.adoc

```diff
@@ -0,0 +1,108 @@
+---
+contentTags:
+  platform:
+  - Cloud
+  - Server v4.x
+  - Server v3.x
+---
+= CircleCI Linux CUDA images support policy
+:page-layout: classic-docs
+:page-liquid:
+:page-description: CircleCI Linux CUDA images release, update, and deprecation policy
+:icons: font
+:toc: macro
+:toc-title:
+
+[#overview]
+== Overview
+
+This document outlines the xref:#[CircleCI Linux CUDA image] release, update, and deprecation policy and applies to our Linux CUDA Images.
+
+[#release-policy]
+== Release policy
+
+The CircleCI CUDA Images are based on our Linux VM machine images for the purpose of providing an image that can leverage the additional processing capabilities of CircleCI’s GPU executor. 
+
+We aim to support the three most recent minor versions (or three most recent even numbered minor versions, if minor version >= `6`) of the two most recent and supported major CUDA releases. We closely track Nvidia’s own release cycle for our releases and aim to release these within a week of the CUDA release. 
+
+Please note that this is not an SLA (service level agreement). We can not, and do not, provide an official SLA turnaround time for new CUDA images.
+
+[#tagging]
+== Tagging
+
+Tagging for these images will be categorized with the following image slugs: 
+
+* `linux-cuda-<CUDA-version>:<year.month.patch>`
+** For example, `linux-cuda-11:2023.02.1`
+** The “default” slugs will simply remove the tag if users want to keep up-to-date:
```

Comment:
@JalexChen I think this needs a bit of clarification or rephrasing. Do we mean to say that the default slug is just the slug without `year.month.patch` ? Which image would that point to? Am I correct in saying that "default" can change without notice?

This is what we have for Linux VM, let me know if it's the same: https://github.com/circleci/circleci-docs/pull/8011/files#diff-788abd596f1e0657f481d530f0bca4d7c3dce21e971ed4e8389aa6c21f711a47R47

Reply:
So, the default tags are meant to cover off on a period of ~2 years, based on CUDA release history, so internally, we would keep the dated tags (up to three, just so there is overlap), but we ultimately want users to _only_  use the default tags, which i think actually need to look like this: `linux-cuda-11:default` or `linux-cuda-12:default`.

what are your thoughts on removing the references to dated tags?

the default tag will always point to the latest version of the image on a rolling basis and no, it will change, but with notice, the idea is that workflows are uninterrupted. there are three components to this:
- every minor/patch release within the major CUDA version is compatible, and we will include up to 3 of the past versions
- there will be an edge image that is created first
- a discuss post will be made letting users know of an update to the versions in the tag

Reply:
@JalexChen I think my preference is to be consistent with the other policy docs, and since this is also a reference type of document we should be detailed and thorough. So I'd still keep the dated tags on here, but would add a recommendation to readers about figuring out which tags work best for their team in the usage guide (e.g. https://circleci.com/docs/using-gpu/).